context-free.md

```diff
@@ -2,21 +2,34 @@
 
 ##  Who is this programming language for?
 
+This language is for artists with a computational bent, or vice-versa.
 
 ## What is easy to do in this language? Why is it easy?
 
+The language is very good at creating recursive images with a lot of shapes in a pattern.
+This means that the images often end up being mathematical and abstract, although that is just a tendency.
+It is good at this because recursive patterns are essentially the main point of the language.
 
 ## What is hard to do in this language? Why is it hard?
 
+The language is not good at creating images that are mostly unpatterned, such as sketches.
```

Comment:
Though it's implied, you didn't really specify why the language can't do this well.